homeassistant/components/notify/tplink_lte.py

```diff
@@ -0,0 +1,57 @@
+"""TP-Link LTE platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.tplink_lte/
+"""
+
+import logging
+
+import voluptuous as vol
+import attr
+
+from homeassistant.components.notify import (
+    BaseNotificationService, ATTR_TARGET, PLATFORM_SCHEMA)
+from homeassistant.const import CONF_HOST
+import homeassistant.helpers.config_validation as cv
+
+from ..tplink_lte import DATA_KEY
+
+
+DEPENDENCIES = ['tplink_lte']
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_HOST): cv.string,
+    vol.Required(ATTR_TARGET): vol.All(cv.ensure_list, [cv.string]),
+})
+
+
+async def async_get_service(hass, config, discovery_info=None):
+    """Get the notification service."""
+    return TplinkNotifyService(hass, config)
+
+
+@attr.s
+class TplinkNotifyService(BaseNotificationService):
+    """Implementation of a notification service."""
+
+    hass = attr.ib()
+    config = attr.ib()
+
+    async def async_send_message(self, message="", **kwargs):
+        """Send a message to a user."""
+        modem_data = self.hass.data[DATA_KEY].get_modem_data(self.config)
+        if not modem_data:
+            _LOGGER.error("No modem available")
+            return
+
+        phone = self.config.get(ATTR_TARGET)
+        targets = kwargs.get(ATTR_TARGET, phone)
+        if targets and message:
+            for target in targets:
+                import tp_connected
```

Comment:
At least move it out of the loop. Usually we put imports in the beginning of the method.

Reply:
You're right, totally missed this

Reply:
@amelchio I'm not skilled at Python so maybe I miss something here. There's any reason why this import is placed inside the loop (that's the same code as `netgear_lte` notify).  I would rather place the import between the line 51 and 52 or the begging of the method as suggested by @MartinHjelmare

Reply:
No reason, @MartinHjelmare is right.